src/python/pants/backend/project_info/cloc.py

```diff
@@ -1,53 +1,49 @@
 # Copyright 2019 Pants project contributors (see CONTRIBUTORS.md).
 # Licensed under the Apache License, Version 2.0 (see LICENSE).
 
-from typing import cast
+from typing import Tuple
 
 from pants.core.util_rules.external_tool import (
     DownloadedExternalTool,
     ExternalTool,
     ExternalToolRequest,
 )
 from pants.engine.console import Console
-from pants.engine.fs import (
-    CreateDigest,
-    Digest,
-    DigestContents,
-    FileContent,
-    MergeDigests,
-    SourcesSnapshot,
-)
+from pants.engine.fs import Digest, MergeDigests, SourcesSnapshot
 from pants.engine.goal import Goal, GoalSubsystem
 from pants.engine.platform import Platform
 from pants.engine.process import Process, ProcessResult
 from pants.engine.rules import Get, collect_rules, goal_rule
+from pants.option.custom_types import shell_str
+from pants.util.enums import match
 from pants.util.logging import LogLevel
 from pants.util.strutil import pluralize
 
 
 class ClocBinary(ExternalTool):
```

Comment:
This should probably be `SccBinary`. In particular, the options scope should change? Even if we keep the goal name the same (and see below for my thoughts there), this models a specific tool, and should be named for that tool.